### ISSUE TITLE
Move to modern .NET 7

### ISSUE THREAD

**Issue by mat1jaczyyy** [mat1jaczyyy/Keyboard-Inspector#24]
Currently we use .NET Framework with Windows Forms which is very ancient, but more importantly locked into Windows.

Moving to cross-platform .NET 7 (likely with Avalonia UI) would be the first step towards Linux and/or macOS support.

**Comment by k557osuman** [comment]
bump this, pls i need native linux support

**Comment by longnguyen2004** [comment]
Just a heads up, my PR is only doing the bare minimum, which is to move to .NET 9 and get it to build. Linux support will require much more than that.

**Comment by mat1jaczyyy** [comment]
Yes, UI rewrite (Avalonia or whatever) and xplat support is way outside the scope of this
